sos/plugins/grafana.py

```diff
@@ -8,16 +8,16 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.plugins import Plugin, RedHatPlugin
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class Grafana(Plugin, RedHatPlugin):
+class Grafana(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     """Fetch Grafana configuration, logs and CLI output
     """
     plugin_name = "grafana"
     profiles = ('services', 'openstack', 'openstack_controller')
 
-    packages = ('graphana',)
+    packages = ('graphana', 'grafana')
```

Comment:
That should be triggered for Xenial as it has grafana offered as a .deb package, but late release seems to only offered grafana as a snap.

`packages =` will only work for RPM and DEB, you'll need something else to trigger the SNAP grafana.

```
$ rmadison grafana
 grafana | 2.6.0+dfsg-1 | xenial/universe | source, amd64, arm64, armhf, i386, ppc64e
```

Reply:
By default the charms use upstream repos to deploy with debs.

Reply:
Ok, thanks for confirming. Would be nice to make that plugin trigger when grafana snap is installed eventually, but I don't see this as a blocker for the time being, it was more a fyi ;)

Reply:
`grafana` is the package name on Red Hat distros and I assume on Debian/Ubuntu as well.

@psachin : where the `graphana` package name comes from, please? Or is it a typo? (shall not be here just `packages = ('grafana')`?)